<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Демодуляция</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #fff;
}
QLabel{
 background-color: #fff;
}
QCheckBox {
  margin-left: 30px;
    spacing: 30px;
}
QCheckBox::indicator {
    width: 30px;
    height: 30px;
	color: blue;
}

QPushButton {
  background-color: #0d6efd;
  color: #fff;
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  outline: 0px;
}

QPushButton#path_button{

  margin-top: 0px;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
}
QListWidget
{
font: normal normal 27px &quot;Arial&quot;;
border : 2px solid black;
background-color: #0d6efd;
}

 QListView::item:selected
{
border : 2px solid black;
background : green;
}

QComboBox{
color: white; 
background-color: #0d6efd;
selection-background-color: lightgreen;
    font: normal normal 27px &quot;Arial&quot;;
}
QComboBox QAbstractItemView {
    background-color: #0d6efd;
    selection-background-color: lightgreen;
    font: normal normal 27px &quot;Arial&quot;;
}
QComboBox QAbstractItemView::item {
    height: 100px;  
    font: normal normal 27px &quot;Arial&quot;;
}
QDoubleSpinBox{
color: white; 
background-color: #0d6efd;
selection-background-color: lightgreen;
    font: normal normal 27px &quot;Arial&quot;;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>792</width>
      <height>495</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>790</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>790</width>
         <height>65</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Настройка параметров демодуляции:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>770</width>
         <height>280</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>770</width>
         <height>280</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>790</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>FM</string>
        </attribute>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>790</width>
           <height>251</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>790</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>790</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>790</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>788</width>
            <height>249</height>
           </rect>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>241</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Частота принимаемого сигнала</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Гц</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_sampl">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentText">
               <string/>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_freq">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Гц</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Частота семплирования</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>NFM</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>WFM</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>AM</string>
        </attribute>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>790</width>
           <height>251</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>790</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>790</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>790</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>788</width>
            <height>249</height>
           </rect>
          </property>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>781</width>
             <height>241</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Частота принимаемого сигнала</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Гц</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_sampl_am">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentText">
               <string/>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_freq_2">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="wrapping">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Гц</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Частота семплирования</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>LSB/USB</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>DSB</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>RAW</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>790</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>790</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Запись сигнала</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="cancel_button_demod">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="continue_button_demod">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Продолжить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
